pkg/apis/serving/v1alpha1/kfservice_framework.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 type FrameworkHandler interface {
-	CreateModelServingContainer(modelName string) *v1.Container
+	CreateModelServingContainer(modelName string, modelServingImageName string) *v1.Container
```

Comment:
I think you should pass the config to the handler. 

@ellis-bigelow the framework handlers already have the spec, so if the value is on the spec the handler already has it..

Reply:
that was my original change, Ellis and I have discussed to avoid having configmap on this interface which is also called on the admission webhook code path where the configmap is not loaded.

Reply:
Is there a reason why configmap cannot be loaded in admission path?

Reply:
*doh* regarding the spec.

We should totally be able to load it from the admission controller.

Reply:
Agreed, can we address this in a separate PR? This requires quite a bit interface change.

Reply:
Yes I think makes sense to do in a separate PR else this will never get done :)

Reply:
I like what I see now.. there are some things that can be tweaked, but I think it makes sense to get this in first and do them in separate PRs

Reply:
/approve